src/services/exportInfoMap.ts

```diff
@@ -422,7 +422,16 @@ export function forEachExternalModuleToImportFrom(
     const autoImportProvider = useAutoImportProvider && host.getPackageJsonAutoImportProvider?.();
     if (autoImportProvider) {
         const start = timestamp();
-        forEachExternalModule(autoImportProvider.getTypeChecker(), autoImportProvider.getSourceFiles(), excludePatterns, (module, file) => cb(module, file, autoImportProvider, /*isFromPackageJson*/ true));
+        const checker = program.getTypeChecker();

```

Comment:
Is there a situation in which this checker is different than `autoImportProvider.getTypeChecker()`? Presumably, given I guess they're different Programs?

Reply:
Yes, they’re always different programs/checkers.